### PR TITLE
Build omes worker image

### PR DESCRIPTION
**Pull request by Sushisource**
Adding omes worker image build to all SDKs

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -216,10 +216,10 @@ jobs:
...
       fail-fast: false
       matrix:
-        node: [16, 18, 20]
-        platform: [linux-x64, linux-arm, macos-x64, macos-arm, windows-x64]
-        reuse-v8-context: [true, false]
-        server: [cli] # FIXME: Add 'cloud'
+        node: [ 16, 18, 20 ]
+        platform: [ linux-x64, linux-arm, macos-x64, macos-arm, windows-x64 ]
+        reuse-v8-context: [ true, false ]
+        server: [ cli ] # FIXME: Add 'cloud'
```
Comment (Sushisource) [inline .github/workflows/ci.yml:222]: My IDE *insists* that this is the proper formatting. I can change it back if anyone thinks it matters.

Comment (mjameswh) [reply .github/workflows/ci.yml]: And mine insists it is not… 🫤

I believe prettier is configured to format that file. Just do `npm run format`, and it should fix what your IDE did.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -580,3 +580,16 @@ jobs:
       #   # Don't run on forks, since secrets won't be available, and command will fail
       #   if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository && github.ref != 'refs/heads/main'
       #   run: npx vercel deploy packages/docs/build -t ${{ secrets.VERCEL_TOKEN }} --name typescript --scope temporal --yes
+
+  omes-image-build:
```
Comment (mjameswh) [inline .github/workflows/ci.yml:584]: Do we want that even for PR branches? Are we expected for this to completes before merging a PR to main?

Comment (Sushisource) [reply .github/workflows/ci.yml:584]: Yes. The idea is to run these per-pr. If it's too slow we can make it optional.

**Review by mjameswh — APPROVED** [review]

**Comment by Sushisource** [comment]
waiting to merge pending decision about pre-or-post merge

**Pull request merged**